src/test.js

```diff
@@ -362,7 +362,7 @@ Test.prototype = {
 	valid: function() {
 		var filter = config.filter,
 			regexFilter = /^(!?)\/([\w\W]*)\/(i?$)/.exec( filter ),
-			module = QUnit.urlParams.module && QUnit.urlParams.module.toLowerCase(),
+			module = QUnit.urlParams.module && QUnit.urlParams.module.toLowerCase(), moduleId,
```

Comment:
would you mind positioning this unnassigned `moduleId` variable to the top line of the var declarations? At least to use a single line for each one.